### PR TITLE
Issue 2013: Increase the timeout in ReadTxnWriteAutoScaleWithFailoverTest, add required logging about  the committed/aborted txns.

### LINKED ISSUE #2013: ReadTxnWriteAutoScaleWithFailover system test  times out while deleting the stream in clean Up phase
**Problem description**
The test times out in clean up phase while it tries to delete the stream.
Additional issues being :
1.  Failover Tests doesn't wait until all the readers are closed before executing clean up phase. and as a 
result we get the following:
```
15 06:36:28,493 717588 [epollEventLoopGroup-4-1] INFO  i.p.c.s.i.AsyncSegmentInputStreamImpl - Closing connection to segment Segment(scope=testReadTxnWriteAutoScaleScope1728405776, streamName=testReadTxnWriteAutoScaleStream, segmentNumber=2) with exception: {}
io.pravega.shared.protocol.netty.ConnectionFailedException: null
```
 2. The information about number of transactions committed/aborted/still in committing stage by the time test times out needs to be added.
**Problem location**
ReadTxnWriteAutoScaleWithFailoverTest.java

**Suggestions for an improvement**
Increase the global timeout, also add logs to know the number of committed, aborted transactions.

### PR DESCRIPTION
**Pull request by Prabha-Veerubhotla**
**Change log description**
1.  Augmenting the timeout to allocate sufficient time for clean up phase.
2.  Adding additional logs which helps in debugging of txns.
3.  Logs related to number of committed/aborted txns.
4.  Keep a note of time of transaction creation and commit.
5.  Waiting for all the readers to be offline before deleting stream/scope.
**Purpose of the change**
Test has less timeout than required. (original timeout = 22 *60s , updated = 25 *60 s)
**What the code does**
Fixes #2013 
**How to verify it**
Run system tests

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -211,6 +211,11 @@ project('segmentstore:storage:impl') {
         testCompile project(':test:testcommon')
         testCompile project(path:':segmentstore:storage', configuration:'testRuntime')
     }
+    test {
```
Comment (fpj) [inline build.gradle]: The changes in the build file do not seem to be related to the issue.

Comment (Prabha-Veerubhotla) [reply build.gradle]: Yeah these are temporary changes , needed for building the image as there are many sporadic unit test failures. Will revert it once the changes are confirmed.

**Comment by Prabha-Veerubhotla** [comment]
it was done in 
https://github.com/pravega/pravega/pull/2113
